src/frontend/packaging_info.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+/*
+As part of a Google Cloud demo, you can run an additional "packaging" microservice (HTTP server).
+This file contains code related to the frontend and the "packaging" microservice.
+*/
+
+type PackagingInfo struct {
+	Weight float32 `json:"weight"`
+	Width  float32 `json:"width"`
+	Height float32 `json:"height"`
+	Depth  float32 `json:"depth"`
+}
+
+func isPackagingServiceConfigured() bool {
+	return os.Getenv("PACKAGING_SERVICE_URL") != ""
```

Comment:
nit: what is a reasoning to read env variable each time this method is called? Is there a way to update the environment variable without restarting the service?

Reply:
I agree that some validation would be ideal.
But I will skip this for now because:
1. For the demo, we might end up using K8s service addresses (`namespace.packagingservice`) instead of a public IP addresses (`http://123.123.123`).
2. We don't do such validation for other environment variables.